benchmarks/busboy/executioner.js

```diff
@@ -0,0 +1,48 @@
+const { process: processBusboy } = require('./contestants/busboy')
+const { process: processFastify } = require('./contestants/fastify-busboy')
+const { getCommonBuilder } = require('../common/commonBuilder')
+const { validateAccuracy } = require('./validator')
+const { resolveContestant } = require('../common/contestantResolver')
+const { outputResults } = require('../common/resultUtils')
+
+const contestants = {
+  busboy: measureBusboy,
+  fastify: measureFastify
+}
+
+async function measureBusboy () {
+  const benchmark = getCommonBuilder()
+    .benchmarkName('Busboy comparison')
+    .benchmarkEntryName('busboy')
+    .asyncFunctionUnderTest(processBusboy)
+    .build()
+  const benchmarkResults = await benchmark.executeAsync()
+  outputResults(benchmark, benchmarkResults)
+}
+
+async function measureFastify () {
+  const benchmark = getCommonBuilder()
+    .benchmarkName('Busboy comparison')
+    .benchmarkEntryName('@fastify/busboy')
```

Comment:
I know you want to name the benchmark like the package, but is as you know a specific character in the filesystem '/'. When the benchmark is finished, it tries to store the results on the filesystem, which results in a ENOENT Error.

Something went wrong: ENOENT: no such file or directory, open '/home/aras/Workspace/fastify/direct/busboy/benchmarks/_results/Busboy_comparison-fastify/busboy-Node_14.json'

Rename it to 
 ```suggestion
    .benchmarkEntryName('@fastify-busboy')
```
and it will work as expected ;)

Reply:
fixed